standard/links/commons/links.lua

```diff
@@ -149,8 +149,9 @@ local PREFIXES = {
 	pubsteam = {'https://steamcommunity.com/groups/'},
 	steamalternative = {'https://steamcommunity.com/profiles/'},
 	stratz = {
-		'',
-		player = 'https://stratz.com/player/'
+		'https://stratz.com/leagues/',
```

Comment:
you sure this doesn't break functionality somewhere?

Reply:
I looked at datdota and dotabuff links and replicated same exact structure:
```
datdota = {
  'https://www.datdota.com/leagues/',
  player = 'https://www.datdota.com/players/',
  team = 'https://www.datdota.com/teams/'
},
```
first item is keyless - a link to league page, then player and team links.

I did not find any any special cases with `dotabuff` or `datdota` keyword in this codebase, so I assume that it should work fine.

Reply:
my point is that if the param was already used expecting no prefix and now it gets the prefix that it might break some links

Reply:
I see!

Reply:
> there are `/dev` versions of the modules change it in them and then use `|dev=1` in the template calls

could you please provide a bit more info? where can I do this?

Reply:
for example the dev module of
https://liquipedia.net/dota2/Module:Infobox/League/Custom
is
https://liquipedia.net/dota2/Module:Infobox/League/Custom/dev

adjust the dev module with what you want to test and use `|dev=1` in the template call^^

Reply:
great! thank you!

new links for league and team pages look good:

![image](https://user-images.githubusercontent.com/20866892/223158492-87207863-3904-4102-9753-19c455d88aff.png)

![image](https://user-images.githubusercontent.com/20866892/223158615-36be19d7-48be-4dd2-8fb4-03551fb16497.png)